qgsquick/from_qgis/plugin/qgsquickfeatureformstyling.qml

```diff
@@ -74,8 +76,8 @@ QtObject {
       property int fontPixelSize: 48 * QgsQuick.Utils.dp
       property real sideMargin: 12 * QgsQuick.Utils.dp // left or right margin for a field's content
       property real outerMargin: 20 * QgsQuick.Utils.dp // left or right margin for a whole field component
-      property int fontPointSize: 16 * QgsQuick.Utils.dp
-      property int labelPointSize: 14 * QgsQuick.Utils.dp
+      property int fontPointSize: 16
+      property int labelPointSize: 14
```

Comment:
True, this should probably not be multiplied by Dp constant, though on Android I feel like the text size is too small then. What if we would rather use pixel sizes? They can be mutliplied and should scale fine on all platforms then. 

<img src="https://user-images.githubusercontent.com/22449698/109834300-532bd780-7c42-11eb-97ac-3214b37ca00d.jpg" width=350>

Reply:
Well, pointSize should be more reliable, but it seems to be not. I think it is not really visible from the second screenshot, but also according some forum threads, pointSize doesn't guarantee uniformity for all devices. I think PeterP was dealing with this at the very beginning and we ended up with pixel sizes and dp multiplier.

So, maybe we can ask Rado to give us also pixelSize for all text formats.